inc/commondbtm.class.php

```diff
@@ -132,12 +132,17 @@ static function getTable($classname = null) {
     * @param string $table name of the table to be forced
     *
     * @return void
-   **/
+    */
    static function forceTable($table) {
       $_SESSION['glpi_table_of'][get_called_class()] = $table;
    }
 
 
+   /**
+    * Retrieve the foreign key of a table related to the called class
+    *
+    * @return mixed
```

Comment:
This function returns a string as it returns result of `getForeignKeyFieldForTable()` function that is stored in cache variable.